### PR TITLE
rename crank to torsiondrive

### PR DESCRIPTION
**Pull request by yudongqiu**

### CODE REVIEW

File: torsiondrive/launch.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 from __future__ import print_function
 
-from crank.DihedralScanner import DihedralScanner
-from crank.QMEngine import EnginePsi4, EngineQChem, EngineTerachem
+from torsiondrive.dihedral_scanner import DihedralScanner
```
Comment (dgasmith) [inline torsiondrive/launch.py:6]: +1 for Python standard module naming.

File: setup.py [hunk 2]
```diff
@@ -8,21 +8,20 @@
...
     description=DOCLINES[0],
     long_description="\n".join(DOCLINES[2:]),
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
-    packages=['crank', "crank.tests"],
+    packages=['torsiondrive', "torsiondrive.tests"],
     entry_points={'console_scripts': [
-        'crank-launch = crank.launch:main',
-        'crank-api = crank.crankAPI:main',
+        'td-launch = torsiondrive.launch:main',
```
Comment (dgasmith) [inline setup.py]: Any reason to shorted this here? `torsiondrive-launch` is very clear and not too long I think.

**Review by dgasmith — APPROVED** [review]
LGTM

File: setup.py (hunk 2 above)
Comment (yudongqiu) [reply setup.py]: Updated 👍

**Pull request merged**